drivers/sensor/lsm303dlhc_accel/Kconfig

```diff
@@ -0,0 +1,53 @@
+# Kconfig - LSM303DLHC Three Axis Accelerometer configuration options
+
+#
+# Copyright (c) 2018 Philémon Jaermann
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig LSM303DLHC_ACCEL
+	bool "LSM303DLHC Three Axis Accelerometer"
+	depends on I2C
+	default n
```

Comment:
Bools are implicitly default n unless specified otherwise, so you can remove this line